bin/style-tools

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+"""
+Usage:
+  style-tools split <tileset> <style>
+  style-tools merge <tileset> <style> <style_header>
+  style-tools recompose <tileset> <style> <style_header>
+  style-tools --help
+  style-tools --version
+
+Actions:
+  split           Break down style.json into separate jsons per yaml layer.
+  merge           Combine separate jsons into style.json.
+
+Options:
+  --help              Show this screen.
+  --version           Show version.
+"""
+from pathlib import Path
+from docopt import docopt
+import openmaptiles
+from openmaptiles.utils import print_err
+from openmaptiles.styleutils import merge, split
+
+if __name__ == '__main__':
```

Comment:
usually you don't want to have a real code inside the `if __name__` statement. Instead, move all the code to the `def main():`, and call that function from the if __name__ ... statement.

Reply:
Moved to `main()` function.